src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -959,3 +959,30 @@
     text: |-
         az openshift wait -g MyResourceGroup -n MyManagedCluster --updated --interval 60 --timeout 1800
 """
+
+helps['openshift monitor'] = """
+type: group
+short-summary: Commands to manage Log Analytics monitoring. Requires "--workspace-id".
+"""
+
+helps['openshift monitor enable'] = """
+type: command
+short-summary: Enable Log Analytics monitoring. Requires "--workspace-id".
+parameters:
+  - name: --workspace-id
```

Comment:
besides, in our guideline, we don't want to have an argument called `workspace-id` to represent ARM id. Is it ok to have `--workspace` and support both name or ARM ID of LA workspace? You can check our guideline.
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#supporting-name-or-id-parameters

Reply:
I am aware of this guideline and we had this discussion for months now:
https://github.com/Azure/azure-cli/pull/10775 and 
https://github.com/Azure/azure-cli/pull/11241
`worskpace-id` is what most people had agreed to and it has been already submitted for `az openshift create` command.